index.d.ts

```diff
@@ -0,0 +1,611 @@
+/* tslint:disable:interface-name max-classes-per-file no-empty-interface */
+
+declare module 'evergreen-ui' {
+  import * as React from 'react'
+  import { EnhancerProps } from 'ui-box/dist/types/enhancers'
+
+  interface UIBoxProps extends Partial<EnhancerProps> {
+    innerRef?: React.Ref<HTMLElement>;
+    children?: React.ReactNode;
+    height?: string | number;
+    width?: string | number;
+    css?: object;
+  }
```

Comment:
I think we might be able to get rid of this by using `BoxProps<T>` for specific components that accept box props in addition to their own props API.

Reply:
Do you mean:
```TS
  export interface CardProps extends PaneProps {
  }

  export class Card extends React.PureComponent<CardProps & BoxProps<'div'>> {
  }
```
 Because this way innerRef is broken and passes `'div' | null` to callback

Reply:
We should fix `innerRef` in ui-box 😅 

https://github.com/segmentio/ui-box/pull/41

Reply:
@Pasalietis there's a new release for ui-box@2.1.2 that has proper types for `innerRef`

Reply:
@mshwery can you look again if now is correct approach for using `UiBox<T>` ?